src/Components/Admin/Clients/Form/index.js

```diff
@@ -107,102 +72,137 @@ function Form() {
     }
   };
 
+  const validate = (formValues) => {
```

Comment:
You should add a validation so that the inputs name and state admit only letters